packages/trader/src/Stores/Modules/Trading/Constants/contract.js

```diff
@@ -133,6 +133,4 @@ export const unsupported_contract_types_list = [
     'lb_call',
     'lb_put',
     'lb_high_low',
-    // TODO: Remove the conditional values below once barrier and path dependent contracts are ready for mobile
-    isMobile() ? 'multiplier' : null,
```

Comment:
Is it fine to remove this line before multiplier responsive release?

Reply:
I think the plan is we do a thorough regression test on a single branch (while keeping in sync with master), then we do a squash merge with all the PRs for multipliers responsive, which will include this check being disabled.